dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -824,3 +824,11 @@ dartsass.compiler.deprecation.warnings.from.dependencies=false
 #analytics.app.write.url=http://jitsu
 #analytics.app.read.url=http://cube
 analytics.accesstoken.renewjob.cron=0 0/1 * * * ?
+
+## By default, We're now removing Content-Types  asynchronously, if you want to go back to the old behavior, set this property to false
+#CONTENT_TYPE_DELETE_ASYNC=true
+
+## By default, Asynchronous Content-Type remove is carried out by a Quartz Job
+## If You want to use the thread pool directly outside the Quartz Job, set this property to false
+## This property is only effective if DELETE_CONTENT_TYPE_ASYNC=true
+DELETE_CONTENT_TYPE_ASYNC_WITH_JOB=false
```

Comment:
This `DELETE_CONTENT_TYPE_ASYNC_WITH_JOB` shouldn't default to `false` to avoid adding a new property to existing dotmarketing-config.properties files coming from previous versions?